packages/extension/src/providers/bitcoin/networks/lbry.ts

```diff
@@ -0,0 +1,51 @@
+import { NetworkNames } from "@enkryptcom/types";
+import {
+  BitcoinNetwork,
+  BitcoinNetworkOptions,
+  PaymentType,
+} from "../types/bitcoin-network";
+import { ssHandler } from "../libs/activity-handlers";
+import wrapActivityHandler from "@/libs/activity-state/wrap-activity-handler";
+import SSFeeHandler from "../libs/ss-fee-handler";
+import SSApi from "../libs/api-ss";
+
+const lbryOptions: BitcoinNetworkOptions = {
+  name: NetworkNames.LBRY,
+  name_long: "LBRY",
+  homePage: "https://lbry.com/",
+  blockExplorerTX: "https://explorer.lbry.com/tx/[[txHash]]",
+  blockExplorerAddr: "https://explorer.lbry.com/address/[[address]]",
+  isTestNetwork: false,
+  currencyName: "LBC",
+  currencyNameLong: "LBRY Credits",
+  icon: require("./icons/lbry.svg"),
+  decimals: 8,
+  node: "https://partners.mewapi.io/nodes/ss/lbc",
+  coingeckoID: "lbry-credits",
+  apiType: SSApi,
+  dust: 0.01,
+  activityHandler: wrapActivityHandler(ssHandler),
```

Comment:
also `ssHandler` is not compatible with getting lbc activity, you can look at ssHandler and create your own activity handler or have a null handler 
https://myetherwallet.gitbook.io/enkrypt-documentation/adding-an-evm-network#adding-activity-handler
this is for evm but the concept is similar